Modix/Modules/UserInfoModule.cs

```diff
@@ -69,7 +69,25 @@ public UserInfoModule(ILogger<UserInfoModule> logger)
                     if (roles.Length > 0)
                     {
                         Array.Sort(roles, StringComparer.OrdinalIgnoreCase);
-                        builder.AppendLine("Roles: " + string.Join(',', (object[])roles));
+                        builder.Append(roles.Length > 1 ? "Roles: " : "Role: ");
+                        for (int i = 0; i < roles.Length; i++)
+                        {
+                            string separator;
+                            if (i == 0)
+                            {
+                                separator = string.Empty;
+                            }
+                            else if (roles.Length > 1 && i + 1 == roles.Length)
+                            {
+                                separator = " and ";
+                            }
+                            else
+                            {
+                                separator = ", ";
+                            }
+                            builder.Append($"{separator}<@&{roles[i].Id}>");
```

Comment:
You should ternary this bitch up.
```csharp
var separator = (i == 0) ? string.Empty
    : ((roles.Length > 1) && ((i + 1) == roles.Length)) ? ", and ";
    : ", ";
```

Reply:
`roles[i].Mention`

Reply:
A little.

Declaring a variable then assigning its value later just makes me cringe, cause it's about 90% of the time a good use for a functional syntax assignment, of some sort.

What you could do is move that to a local function, so you can make use of if/return syntax.
```csharp
if(...)
    return ...;
else if(...)
    return ...:
else
    return ...;
```

Or just leave it as is. Not a big deal.

Supposedly, we might get a switch/case style expression soon, that would work well here. E.G.
```csharp
var separator = switch
{
    case (i == 0):                                        string.Empty;
    case (roles.Length > 2) && (i == (roles.Length - 1)): ", and ";
    case (roles.Length > 1) && (i == (roles.Length - 1)): " and ";
    default:                                              ", ";
}
```